### ISSUE TITLE
Bug: Missing passClick.png in /images directory

### ISSUE THREAD

**Issue by hellconnon** [kc-ml2/llvm-flow#3]
Thank you for your application, i have a minor issue, that occurs when following the docker compose guide. It seems one resource is not included within the github repo.

```ts
import passClick from '@/images/passClick.png'
```

The resource used in this import statement is not present within the /src/images directory and causes webpack to throw an error.

```sh
frontend-1  | ERROR in ./src/components/pages/tutorial/Tutorial.tsx 17:0-47
frontend-1  | Module not found: Error: Can't resolve '@/images/passClick.png' in '/llvm-flow-frontend/src/components/pages/tutorial'
frontend-1  | 
frontend-1  | webpack compiled with 1 error and 1 warning
frontend-1  | No issues found.
Gracefully stopping... (press Ctrl+C again to force)
```

This bug is present in the latest master version as well as the latest tagged version.

To reproduce just clone the version of the repo and follow the install guide, when accessing the web ui via http://localhost:3000 an error screen should occur and the docker logs should show the error pasted above.

**Comment by ML2jinmyoung** [comment]
Thanks for reporting this issue and for the clear reproduction steps!
You're absolutely right — the image passClick.png was missing from the /src/images directory. I've added the missing resource, and the error should no longer occur when following the Docker Compose setup.

**Please pull the latest version and let me know if everything works as expected now** 🙌
Appreciate your help in making this project better!

**Comment by hellconnon** [comment]
Wow, this was fast, did not see it. Then sorry for my PR. Thanks

**Comment by hellconnon** [comment]
I can confirm that the application now starts as expected.
